runtime/src/main/java/org/corfudb/runtime/collections/CorfuTable.java

```diff
@@ -507,6 +517,15 @@ protected void undoRemove(CorfuTable<K, V> table, V undoRecord, K key) {
         }
     }
 
+    @DontInstrument
+    protected List<Object> identifyRemoveGarbage(Object locator, K key) {
+        List<Object> garbage = new ArrayList<>();
+
+        // TODO(Xin): Distinguish put and remove in the future
+        garbage.addAll(locatorStore.addUnsafe(key, (SMRRecordLocator) locator));
```

Comment:
Same here, you can use `List<Object> garbage = new ArrayList<>(locatorStore.addUnsafe(key, (SMRRecordLocator) locator));`

Reply:
done.